### ISSUE TITLE
Generate `banner` in dist files

### ISSUE THREAD

**Issue by nicolaisueper** [micromata/bootstrap-kickstart#182]
We put a banner on top of every source file in earlier versions of Bootstrap Kickstart:

```
/*! <%= pkg.title %> - v<%= pkg.version %>\n +
 * <%= pkg.author.email %>\n +
 * Copyright ©<%= grunt.template.today("yyyy") %> <%= pkg.author.name %>\n +
 * <%= grunt.template.today("yyyy-mm-dd") %>\n +
 */
```

Do we still need this option? For our use cases it isn't needed usually. Can I get an upvote (👍) or downvote (👎) for reimplementing this feature into our new Gulp workflow @micromata/ux?

**Comment by mischah** [comment]
We don’t need that cause we disabled banners in production because of »security« reasons.
See #27

If we decide to bring em back it should be a »global« setting in `gulp/config.js`.

**Comment by lfuelling** [comment]
+1 for the optional way

**Comment by mischah** [comment]
Reopen because there is one who wants it.

@nicolaisueper takes care of implementing it. 

Like said before: This is optional and defaults to `useBanners = false`.

**Comment by mischah** [comment]
Hej @nicolaisueper, what’s the status of this? 😘 

I’d like to prepare the release of Baumeister v1.0.0
Should we ship this featuré with Baumeister 1.1.0?

**Comment by nicolaisueper** [comment]
Hey @mischah, sorry for the late answer. I was quite busy the last days.
I stopped working on it after I tried to `gulp.src` and `gulp.dest` multiple source directories. I found a way for gulp 3, but that does not work anymore. Going to research later. I also need to fix your requested changes in the v4 PR of the generator. :)

Maybe I get this done today/tomorrow. Otherwise we should ship it with a later version.
